### DISCUSSION TITLE
combined LXC for *arr packages?

### DISCUSSION THREAD

**Discussion by chenks** [tteck/Proxmox#3399] [Ideas]
has there been any thought about creating a LXC that runs all the *arr packages in one LXC?
as they all use different ports i can't see there would any issues?

as it stands just now you would need to create an LXC for each of the individual *arr packages, but having them all in the 1 LXC would be a cleaner solution? the script could ask which app you want to have installed as part of the setup process?

**Comment by Mellowlynx** [comment]
Check out the MediaStack project, maybe something you can use in a docker LXC.
https://github.com/geekau/mediastack

**Reply by chenks** [reply]
I know it can be done in docker, but that just adds another layer which isn't necessary when they can be run natively inside an LXC
